### ISSUE TITLE
Questions about performance degradation and configurations

### ISSUE THREAD

**Issue by AMShek** [yoonkicho/PPLR#11]
Thank you for releasing codes for your interesting work!

I have run the training on 4 GPUs with the default configurations (directly running the commands you give in ReadMe), but **the results are much lower than the performance stated in your paper, shown as follow.**

Is this because the configurations are not the same with your experiments? If so, **could you please also release configurations files** so that we can more easily follow your great work. 

- Results on Market 
![market](https://user-images.githubusercontent.com/26130484/200729993-e8ea8996-4f53-4f8b-9e28-c018d51bd343.png)

- Results on Market with cam ID

![market_cam](https://user-images.githubusercontent.com/26130484/200730003-b45312b1-bd3f-45f2-a49b-6c1daa21ab23.png)

- Results on MSMT

![msmt](https://user-images.githubusercontent.com/26130484/200730015-f5f0c70f-6760-486e-93dc-58313f1be941.png)

**Comment by yoonkicho** [comment]
Hi, thank your for your interest.
We used the same configuration with the released code.
Unsupervised re-ID training showed different performances each time despite training with a fixed seed.
"Setting --eval-step to 1" could give more similar results to the paper.
We used 4 TITAN RTX GPUs with PyTorch==1.6.0 and torchvision==0.7.0.
